src/components/header/header.spec.js

```diff
@@ -0,0 +1,27 @@
+import { expect } from '@esm-bundle/chai';
+import './header.js';
+
+describe('Components/Header', () => {
+  let header;
+
+  before(async () => {
+    header = document.createElement('tt-header');
+
+    document.body.appendChild(header);
+
+    await header.updateComplete;
+  });
+
+  describe('Default Behavior', () => {
+    it('should not be null', () => {
+      expect(header).not.equal(undefined);
+      expect(header.querySelectorAll('header').length).equal(1);
```

Comment:
Maybe a quick test here to see if at least one `<img>` exists and it's `src` and `alt` attributes are not undefined?

Reply:
sure. I am not familiar with chai and was trying to get at least the minimum testing in place to be expanded on.